### DISCUSSION TITLE
How can I prevent parent element animation on whileTap at children element

### DISCUSSION THREAD

**Discussion by monodyle** [motiondivision/motion#2156] [Q&A]
Hi guys, I have an item that has an animation while tapping on it. And there's also a children element have whileTap trigger too.

But how can I prevent parents' animation when tapping on the children?

Code example: https://codesandbox.io/s/framer-motion-parent-and-children-fo14wy?file=/src/lib/card.tsx

![image](https://github.com/framer/motion/assets/30283022/3f12a639-f3de-4531-a590-123448fb8c78)

**Comment by MuhammadYousufHere** [comment]
you could use conditional - using state.
let's say if you click over contaiener pass the whileTap state and vice versa 
(PS: there could be better option)
